docs/1-introduction/1-1-getting-started/1-1-2-hello-d3d11.md

```diff
@@ -1,8 +1,19 @@
 # Hello D3D11
 
-We setup a base application in the former chapter. We wont be showing `Main.cpp` and `Application.cpp` anymore.
-
-Let's break down the relevant bits and pieces by showing you how the new class, which derives from `Application` will look like.
+In this chapter, we'll introduce you to the basics of using D3D11; how to create a ID3D11Device and
+how to use it to show something in our window. In the last chapter we setup a basic implementation
+for a application with a window through GLFW. The implementation for `Main.cpp` and
+`Application.cpp` won't be shown here anymore.
+
+If you are looking at the source code for this chapter, you will also notice that
+`Application.cpp` and `Application.hpp` do not exist anymore, as we have moved both of these
+files into a separate `Framework` target to ease development between chapters. This `Framework`
+target will include code that is shared between all chapters, so it might include a lot of other
+files which are not used or are not relevant within some chapter. Please note that the code for
```

Comment:
`chapters`

Reply:
Not necessarily. Both works, but I'll change it.